tests/cases/case_care_platform_participants.yaml

```diff
@@ -80,4 +80,93 @@ TestCreateFollowUpTicket:
       Var: TODAYS_DATE
     - Type: wait_for
       Strategy: xpath
-      Id: $PAGE.care_platform_participant_details.new_ticket_date$
\ No newline at end of file
+      Id: $PAGE.care_platform_participant_details.new_ticket_date$
+
+#CXTB-251 Access Participant Directory and Filter
+TestParticipantDirectoryFilter:
+  Environment: Settings 
+  Roles:
+    - Role: desktopChrome
+      App: "desktop"
+  Actions:
+  #Calling Login case
+    - Type: case
+      Value: CarePartnerLogin
+  #Go to participants
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform.navigation_participant$
+  #Get the value to compare later
+    - Type: get_text
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant.first_darkened_letter_button$
+      Greps:
+        - var: LASTNAME_FILTER_LETTER
+          match: "(.*)"
+  #Click the first darkened letter button
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant.first_darkened_letter_button$
+  #Verify it was filtered by the selected letter
+    - Type: get_text
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant.first_senior_in_list_strong$
```

Comment:
What is this "strong" at the end of each list?

Reply:
The etiquette that contains the lastname is a Strong etiquette. Each participant row has a Strong etiquette. You can also check it on the page object, that it is calling the strong on the first row of the page.